tests/scripts/run_tests.sh

```diff
@@ -8,7 +8,7 @@ set -ex
 # delete any __pycache__ folders to avoid hard-to-debug caching issues
 find . -name __pycache__ -type d -exec rm -r '{}' + || true
 PYTEST_ARGS=("${PYTEST_ARGS}")
-py.test -vv "${PYTEST_ARGS[*]}" "${PYTEST_JUNIT}" tests
+py.test -vv "${PYTEST_ARGS[*]}" "${PYTEST_JUNIT}" tests -s
```

Comment:
I am leaving the `-s` flag here so we can see the logs when `make coverage` runs. Otherwise, if the tests get stuck again, we have no way to know what is stopping them.